Directory.Packages.props

```diff
@@ -3,11 +3,13 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
     <AccessToNugetFeed>true</AccessToNugetFeed>
+    <ToolingPackagesVersion>1.1.1.3883</ToolingPackagesVersion>
   </PropertyGroup>
   <ItemGroup Condition="$(AccessToNugetFeed)">
-    <PackageVersion Include="ContentFeedNuget" Version="1.1.0-3867" />
+    <PackageVersion Include="ContentFeedNuget" Version="$(ToolingPackagesVersion)" />
```

Comment:
Doesn't this break dependabot updates?

Reply:
I don't actually think so, I had something in another project (can't remember where) where this seemed to work. But I guess we will find out? 

If it doesn't I'll do dependabot grouped updates, and add a msbuild flag to fail the build if these get out of sync